### PR TITLE
Fix issue #773 : Test should not be skipped when the exception is expected

### PR DESCRIPTION
**Pull request by juherr**

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -2,6 +2,7 @@ Current
 
 New: GITHUB-717: Add assertThrows and expectThrows (Ryan Schmitt)
 Fixed: GITHUB-755: Fixed reporting of retried tests (Ryan Schmitt)
+Fixed: GITHUB-773: Test should be skipped when the exception is expected (@CandyLiuM & Julien Herr)
```
Comment (cbeust) [inline CHANGES.txt]: You probably mean "not be skipped" here.

**Comment by juherr** [comment]
Yes. I changed the commit message but forgot the file. I will fix it.

File: CHANGES.txt (hunk 1 above)
Comment (juherr) [inline CHANGES.txt]: Yes. I changed the commit message but forgot the file. I will fix it.

Done.

**Comment by juherr** [comment]
Ping @cbeust

**Pull request merged**